jscomp/core/lam_compile_external_obj.ml

```diff
@@ -69,7 +69,7 @@ let assemble_obj_args (labels : External_arg_spec.obj_params)
         match acc with
         | Splice2 _ | Splice0 -> assert false
         | Splice1 x ->
-            ((Js_op.Lit label, x) :: accs, Ext_list.append new_eff eff, assign)
```

Comment:
Is the goal to get rid of `Ext_list`?

Reply:
I regret doing this mixed with the other changes, but I removed some functions that existed in `List.` for the OCaml versions that we support.